go/exocom/src/security/manager.go

```diff
@@ -33,6 +36,17 @@ func (m *Manager) ReceiveMessage(message structs.Message) *Result {
 	var messageToSend *structs.Message
 	warningMessage := ""
 	switch message.Name {
+	case "security request":
+		bytes, err := json.Marshal(message.Payload)
+		if err != nil {
+			warningMessage = err.Error()
```

Comment:
Please wrap these errors with https://github.com/pkg/errors to give more context. For example here
`warningMessage = errors.Wrap(err, "Marshaling 'security request' payload").Error()`